tests/common/devices.py

```diff
@@ -1138,6 +1138,22 @@ def get_asic_name(self):
 
         return asic
 
+    def get_running_config_facts(self):
+        return self.config_facts(host=self.hostname, source='running')['ansible_facts']
+
+    def get_vlan_intfs(self):
+        '''
+        Get any interfaces belonging to a VLAN
+        '''
+        vlan_members_facts = self.get_running_config_facts()['VLAN_MEMBER']
+        vlan_intfs = []
+
+        for vlan in vlan_members_facts:
+            for intf in vlan_members_facts[vlan]:
+                vlan_intfs.append(intf)
+
+        return list(sorted(vlan_intfs, key=lambda intf: int(intf.replace('Ethernet', ''))))
```

Comment:
Here you can simply return the `vlan_intfs` list untouched. Sorting and replacing the port names with port index may not be a common requirement.